core/src/main/java/com/linecorp/armeria/server/cors/CorsPolicy.java

```diff
@@ -363,17 +365,17 @@ static String toString(Object obj, @Nullable Set<String> origins, List<Route> ro
                            @Nullable Set<AsciiString> allowedRequestHeaders,
                            @Nullable Map<AsciiString, Supplier<?>> preflightResponseHeaders) {
         return MoreObjects.toStringHelper(obj)
-                          .omitNullValues()
-                          .add("origins", origins)
-                          .add("routes", routes)
-                          .add("nullOriginAllowed", nullOriginAllowed)
-                          .add("credentialsAllowed", credentialsAllowed)
-                          .add("maxAge", maxAge)
-                          .add("exposedHeaders", exposedHeaders)
-                          .add("allowedRequestMethods", allowedRequestMethods)
-                          .add("allowAllRequestHeaders", allowAllRequestHeaders)
-                          .add("allowedRequestHeaders", allowedRequestHeaders)
-                          .add("preflightResponseHeaders", preflightResponseHeaders).toString();
+                .omitNullValues()
+                .add("origins", origins)
+                .add("routes", routes)
```

Comment:
We could revert this indentation changes because it doesn't actually change anything.